Formula/athena-cli.rb

```diff
@@ -0,0 +1,132 @@
+class AthenaCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "CLI tool for AWS Athena service"
+  homepage "https://athenacli.readthedocs.io/en/latest/"
+  url "https://files.pythonhosted.org/packages/7b/d8/8e3525213c209cb6de70ef9074fe356bada9e8871b6b85fcd860d571071b/athenacli-1.4.1.tar.gz"
+  sha256 "ad70993238d718a8eae5b573b12af83574ceb85ae694daa5c2e98215f89a5423"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.8"
+
+  # Generated with homebrew-pypi-poet from:
+  # poet -r athenacli \
```

Comment:
`brew update-python-resources` doesn't work?

Reply:
oh, I did not try that route. I can remove this to avoid confusion for the future.